.dockerignore

```diff
@@ -1,9 +0,0 @@
-.git
```

Comment:
This should not be removed, it should be rather renamed to `.s2iignore` and used to shrink the image footprint.

Reply:
^ meaning the whole `.dockerignore` file, not the `.git` line only.

Reply:
See https://github.com/openshift/source-to-image#build-workflow 

![image](https://user-images.githubusercontent.com/7453394/175928293-4eeea678-04da-4ad1-a29a-76d86eea23d6.png)

Reply:
Added in the latest commit :+1: